etc/dbus-serialbattery/battery.py

```diff
@@ -1094,6 +1100,19 @@ def log_settings(self) -> None:
         logger.info(
             f"> CCCM SOC: {str(utils.CCCM_SOC_ENABLE).ljust(5)} | DCCM SOC: {utils.DCCM_SOC_ENABLE}"
         )
+        if utils.AUTO_RESET_SOC and utils.JK_BMS_AUTO_RESET_SOC:
```

Comment:
Why are you using two settings? This makes the configuration much more complicated for nothing.